Lib/shutil.py

```diff
@@ -384,8 +384,8 @@ def copy(src, dst, *, follow_symlinks=True):
     return dst
 
 def copy2(src, dst, *, follow_symlinks=True):
-    """Copy data and all stat info ("cp -p src dst"). Return the file's
-    destination.
+    """Copy data and stat info (mode bits, mtime, atime, flags). Return the file's
```

Comment:
I'm confused by "mode bits", I would prefer to only write "mode". I don't know what are "flags". I'm unable to find "st_flags" in os.stat_result.

I suggest to either give the full list of all copied attributes, or document which ones are not copied. But this documentation seems to be half baked.

Reply:
I have made the requested changes; please review again.

I've changed the rst. I think it is written well mostly, so only the text 'all' has been replaced to 'more', which is correct now I believe.

'mode bits' has been replaced everywhere to 'mode' as you suggested. Also, I decided to write the concrete list of what metadata is copied by copystat, and copy2() refers to it to avoid unnecessary duplicating of the docstring.

st_flags is actually returned by stat() on some systems (*bsd and osx I believe), so I kept it there (I think it is not in stat_info's repr()), with the notation that flags is only copied when os.chflags() is available and st_flags is returned by stat(). More info about chflags: https://www.freebsd.org/cgi/man.cgi?query=chflags